### ISSUE TITLE
Issue with Google Calendar sync

### ISSUE THREAD

**Issue by Majkel97** [calcom/cal.com#8376]
### Issue Summary

Hello, we have a problem with Google Calendar synchronization on our self-hosted app. If someone connects Google Calendar to an account on the app, events are synced, but future changes (new events, rescheduling, or deletion) directly in google calendar are not moved to cal.com app. To see changes, users have to delete and connect calendars one more time.

### Steps to Reproduce

1. Create an account on cal.com app.
2. Add Google Calendar to the account.
3. Check the booking page - all is fine.
4. Make changes in Google Calendar (add / edit / delete events).
5. Check the booking page one more time - we have the same state as at point 3.
6. Delete the calendar and add one more time.
7. Now data on the booking page are properly.

App version: 2.8.1

**Comment by Tom-Potanski** [comment]
Yes, we have the same issue. Hope the developers at cal.com can fix it. Really bad the calendar does not synchronize. Makes me think of switching back to calendly.

**Comment by pawanssingh456** [comment]
I would like to work on this issue as I have extensive knowledge of Google Calendar API

Here are a few questions:-

1. Are we using a webhook response from Google to update the calendar data?
2. Are we using poll data to update the calendar data and what is polling frequency?

**Comment by SimonBiggs** [comment]
@Carpea03 seems like others are also having this issue.

@Majkel97, do you happen to know a version where syncing did work? For example, a version we might be able to roll our instance back to?

**Comment by PeerRich** [comment]
is this only on self-hosting? maybe something is wrong with your scope of permissions.

can someone confirm this is on app.cal.com too?

**Comment by SimonBiggs** [comment]
> can someone confirm this is on app.cal.com too?

This doesn't appear to occur on app.cal.com. Only the self hosted version. Let me know if you'd like me to provide any further information to help you reproduce the issue.

**Comment by SimonBiggs** [comment]
Thanks @roae and @zomars!! I can confirm that this is indeed fixed!

Thank you! :tada:

**Comment by fayre-s** [comment]
Hi
Think this issue might have recurred - on the app Cal.com

My Cal.com is not syncing with my google calendar. I have noticed it because I use Cal.com for work and have received a booking for a time when I have a clash on my calendar (with certainty). I have been through all my setttings on Cal.com and calendar and can't see why this could have happened, other than the Cal.com app not syncing.

Any suggestions?

**Comment by mc25573** [comment]
I'm also having an issue with google calendar. When a meeting is scheduled, it only shows up in cal.com. No emails are sent and no event is created in google calendar. I have the google calendar app installed in cal.com. Unfortunately this means I have to switch to calendly.

**Comment by imadamblack** [comment]
Hey, I'm having this issue as well.

I'm using the web version of cal.com and every time I change an event from my Google calendar (update or delete) it stays the same in the cal.com platform, the main issue here is that the time slot remains blocked.

Have you find a solution?

**Comment by timstoenner** [comment]
I am experiencing this issue today! Please update on solution - this should be a core, monitored functionality of the app.

**Comment by JuanxCursed** [comment]
same issue here

**Comment by ASTRAAdvertising** [comment]
Still not resolved...

**Comment by cms-aykutaskeri** [comment]
+1
